tests/e2e/run-e2e-test.sh

```diff
@@ -149,6 +149,7 @@ function usage() {
   echo "  --auth-proxy                                 use to run e2e auth-proxy suite"
   echo "  --resiliency                                 use to run e2e resiliency suite"
   echo "  --app-mobility                               use to run e2e application-mobility suite"
+  echo "  --no-modules                                 use to run e2e powerflex suite without any modules"
```

Comment:
Should this be generalized for all drivers, not just powerflex?

Reply:
Resolved in latest commit